src/main/java/com/angelbroking/smartapi/smartstream/ticker/SmartStreamTicker.java

```diff
@@ -65,6 +68,30 @@ public SmartStreamTicker(String clientId, String feedToken, SmartStreamListener
         init();
     }
 
+	/**
+	 * Initializes the SmartStreamTicker.
+	 *
+	 * @param clientId            - the client ID used for authentication
+	 * @param feedToken           - the feed token used for authentication
+	 * @param delay               - delay in milliseconds
+	 * @param period              - period in milliseconds
+	 * @param smartStreamListener - the SmartStreamListener for receiving callbacks
+	 * @throws IllegalArgumentException - if the clientId, feedToken, or SmartStreamListener is null or empty
+	 */
+	public SmartStreamTicker(String clientId, String feedToken, SmartStreamListener smartStreamListener, Integer delay, Integer period ) {
+		if (Utils.isEmpty(clientId) || Utils.isEmpty(feedToken) || Utils.isEmpty(delay) || Utils.isEmpty(period) ||  Utils.validateInputNullCheck(smartStreamListener)) {

```

Comment:
use stringutils.isempty not custom function?

Reply:
using isEmpty() of String class now to check whether the given string is empty or not.

Reply:
Now using StringUtils.isEmpty() which will check for both null and empty value for string .